### DISCUSSION TITLE
Need help integrating self-hosted MediaFusion with Prowlarr

### DISCUSSION THREAD

**Discussion by xgCommander** [mhdzumair/MediaFusion#356] [Q&A]
I need some info to understand a few things regarding the Prowlarr setup.

**My current setup (completely self-hosted):**
I request movies in Radarr -> Radarr send a request to Prowlarr -> Prowlarr gets the torrent from the indexers I've added -> sends the torrent to QBittorrent -> Radarr imports the movie once it's downloaded

**What I've done to setup MediaFusion:**
1. I downloaded the docker-compose file, removed Prowlarr and Nginx (since I already use Traefik) and deployed it
2. I got the Prowlarr indexer file (resources/yaml/mediafusion.yaml) file and imported it into Prowlarr
3. I added the MediaFusion indexer and provided the manifest URL I copied from the self-hosted MediaFusion instance

**The issue I'm having:**
I saw that the base URL was pointing to https://mediafusion.elfhosted.com, so I modified the resources/yaml/mediafusion.yaml and added the local MediaFusion link as well but its not working (Query ran but no results). If I use https://mediafusion.elfhosted.com, it works but I get rate limited sometimes.

As a self-hosting enthusiast, I would love the get the self-hosted version working but I feel like I messed up the setup somewhere or maybe I'm doing it all wrong.

@mhdzumair can you tell me what I've done wrong and how I change it so that it works with my existing setup?
Also, I would love to understand how this application works at a very high level.

**Comment by kushvinth** [comment]
+1
